packages/data-bite/README.md

```diff
@@ -26,6 +26,13 @@ export default App;
 
 Note, you must pass in a configuration object. Configuration objects can be created with the [Editor package](https://github.com/CDCgov/cdc-open-viz/tree/main/packages/editor).
 
+```
+sampleConfig = {
```

Comment:
Again, probably can link to the /examples/ file here

Reply:
This has been updated.